src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,71 @@
+package calculator;
+
+public class StringCalculator {
+
+    private static final String SUPPORT_OPERATOR = "+-*/";
+    private static final String DELIMITER = " ";
+
+    public int calculate(String input) {
+        validateInput(input);
+
+        String[] splitInput = input.split(" ");
+
+        int answer = Integer.parseInt(splitInput[0]);
+        for (int i = 1; i < splitInput.length; i += 2) {
+            String operator = splitInput[i];
+            int number = Integer.parseInt(splitInput[i + 1]);
+
+            validateOperator(operator);
+
+            if (operator.equals("+")) {
+                answer = add(answer, number);
+            }
+            if (operator.equals("-")) {
+                answer = subtract(answer, number);
+            }
+            if (operator.equals("*")) {
+                answer = multiply(answer, number);
+            }
+            if (operator.equals("/")) {
+                answer = division(answer, number);
+            }
+        }
+
+        return answer;
+    }
+
+    private void validateInput(String input) {
+        if (input == null || input.isEmpty() || input.trim().isEmpty()) {
```

Comment:
input.isEmpty() || input.trim().isEmpty() 조건은 줄여도 될 것 같아요,
```suggestion
        if (input == null || input.trim().isEmpty()) {
```

Reply:
넵 리뷰 감사드립니다
7e00d8c12ce97733e133733e973eac439a7aa90c 커밋에 반영하였습니다